CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.0 (2019-07-26)
+
+- Added `activeIndex` property to repeater component to indicate which repeater item shows an active state. [#51](https://github.com/blackbaud/skyux-lists/pull/51)
```

Comment:
Added `activeIndex` to the repeater component properties to indicate which repeater item shows an active state.

Reply:
Done!